content/contentfeed.inc

```diff
@@ -133,51 +132,65 @@ if ($ncommentResults) {
   }
 }
 
-##REFERENCE ITEMS##
-$referenceSQL = "SELECT revision.rev_id,revision.rev_timestamp,user.user_name, page.page_title, revision.rev_page
-  FROM revision
-  JOIN user
-  ON revision.rev_user=user.user_id
-  JOIN page
-  ON revision.rev_page=page.page_id
-  WHERE rev_minor_edit=0
-  AND page.page_namespace=0
-  and rev_timestamp > $cutoffWiki
-  ORDER BY rev_timestamp DESC";
-$referenceResults = dbQuery("cmubuggy_wiki",$referenceSQL);
-
-$referenceClusterItems = array();
-
-if ($referenceResults) {
-  $icon_map["reference"] = '<span class="fa-layers fa-fw fa-2x mr-2"><i class="fas fa-square text-muted"></i><i class="fas fa-info fa-inverse" data-fa-transform="shrink-8"></i></span>';
-  while ($post = mysqli_fetch_row($referenceResults)) {
-    $timestamp = $post[1];
+// smugmug comments
+$smugMugCommentSQL = "SELECT comment_url, author, created_at
+                      FROM smugmug_comments
+                      WHERE created_at > '$cutoffDate'
+                      ORDER BY created_at DESC;";
+$smugMugCommentResults = dbQuery($HISTORY_DATABASE, $smugMugCommentSQL);
+
+if ($smugMugCommentResults) {
+  $icon_map["icomment"] = '<span class="fa-layers fa-fw fa-2x mr-2"><i class="fas fa-square text-muted"></i><i class="fas fa-image fa-inverse" data-fa-transform="shrink-8"></i><i class="fas fa-comment" data-fa-transform="shrink-10 right-6 down-6"></i></span>';
+  while($row = $smugMugCommentResults->fetch_assoc()) {
+    $timestamp = $row["created_at"];
     $year = substr($timestamp, 0, 4);
-    $month = substr($timestamp, 4, 2);
-    $day = substr($timestamp, 6, 2);
-    $hour = substr($timestamp, 8, 2);
-    $minute = substr($timestamp, 10, 2);
-    $second = substr($timestamp, 12, 2);
-    $timestamp = mktime($hour, $minute, $second, $month, $day, $year)+$UTCdiff;
-
-    $href = "/reference/".$post[3];
-    $clusterKey = $post[4].$year.$month.$day;
-    if (array_key_exists($clusterKey, $referenceClusterItems)) {
-      $referenceClusterItems[$clusterKey]["count"]++;
-      $referenceClusterItems[$clusterKey]["content"] = $referenceClusterItems[$clusterKey]["count"] . ' changes to <a href="' . $href . '">' . str_replace("_"," ",$post[3]) . '</a>';
-    } else {
-      $referenceClusterItems[$clusterKey] = array(
-        "timestamp"=>$timestamp,
-        "count"=>1,
-        "content"=>$post[2] . ' edited <a href="' . $href . '">' . str_replace("_"," ",$post[3]) . '</a>',
-        "type"=>"reference"
-      );
-    }
+    $month = substr($timestamp, 5, 2);
+    $day = substr($timestamp, 8, 2);
+    $hour = substr($timestamp, 11, 2);
+    $minute = substr($timestamp, 14, 2);
+    $second = substr($timestamp, 17, 2);
+    $timestamp = gmmktime($hour, $minute, $second, $month, $day, $year);
+
+    $nextItem = array(
+      "timestamp" => $timestamp,
+      "content" => $row["author"] . ' <a href="' . $row["comment_url"] . '">commented on</a> an image',
```

Comment:
In theory to make it a little less awkward that it doesn't open the comments, you could link "image" instead. No preference, but just a thought.

Reply:
Yeah, that makes sense even if it makes me sad, done.